lib/client.js

```diff
@@ -402,57 +402,57 @@ class HTTPParserError extends Error {
   }
 }
 
+let currentParser = null
+const llhttp = new WebAssembly.Instance(mod, {
```

Comment:
I think here we can specify memory limits, if we need it in the future

Reply:
Do we know what the default limits are?

Reply:
I couldn't find it yet. Maybe a manual inspection could tell. But it should be easy to pass a custom memory instance, we just need to add a `memory` key to `env` if I am not mistaken:

* https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WebAssembly/Memory/Memory#parameters
* https://ariya.io/2019/05/basics-of-memory-access-in-webassembly

Btw, I am not saying we should do it now.